Makefile

```diff
@@ -12,60 +16,63 @@ ifdef TEST_MODEL
 MODEL_COMMAND = models/$(TEST_MODEL)
 endif
 
-@PHONEY: clean
+.PHONY: clean
 clean:
 	mkdir -p dist
 	rm -r dist
 	rm -f baked_version
 
-@PHONEY: build
+.PHONY: build
 build: clean
```

Comment:
Would it make sense for the `build` target to depend on the `create-version` target?